Source/TapjoyAdapter.swift

```diff
@@ -80,33 +80,31 @@ final class TapjoyAdapter: PartnerAdapter {
     /// - parameter status: One of the `GDPRConsentStatus` values depending on the user's preference.
     func setGDPR(applies: Bool?, status: GDPRConsentStatus) {
         // See https://dev.tapjoy.com/en/ios-sdk/User-Privacy
-        let policy = Tapjoy.getPrivacyPolicy()
-        if let applies = applies {
-            policy.setSubjectToGDPR(applies)
-            log(.privacyUpdated(setting: "setSubjectToGDPR", value: applies))
-        }
-        if status != .unknown {
-            let userConsent = status == .granted ? "1" : "0"
-            log(.privacyUpdated(setting: "userConsent", value: userConsent))
-            policy.setUserConsent(userConsent)
-        }
+        let appliesStatus = TJStatus(value: applies)
+        Tapjoy.getPrivacyPolicy().subjectToGDPRStatus = appliesStatus
+        log(.privacyUpdated(setting: "subjectToGDPRStatus", value: applies))
+
+        let consentStatus = TJStatus(value: status)
+        log(.privacyUpdated(setting: "userConsentStatus", value: status))
+        Tapjoy.getPrivacyPolicy().userConsentStatus = consentStatus
```

Comment:
```suggestion
        let consentStatus = TJStatus(value: status)
        Tapjoy.getPrivacyPolicy().userConsentStatus = consentStatus
        log(.privacyUpdated(setting: "userConsentStatus", value: status))
```
Nit to keep the order consistent.